src/getCoinCombination.test.js

```diff
@@ -7,4 +7,52 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return [0, 0, 0, 0] for 0 cents', () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
+
+  it('should return [1, 0, 0, 0] for 1 cent', () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+  });
+
+  it('should return [0, 1, 0, 0] for 5 cents', () => {
+    expect(getCoinCombination(5)).toEqual([0, 1, 0, 0]);
+  });
+
+  it('should return [0, 0, 1, 0] for 10 cents', () => {
+    expect(getCoinCombination(10)).toEqual([0, 0, 1, 0]);
+  });
+
+  it('should return [0, 0, 0, 1] for 25 cents', () => {
+    expect(getCoinCombination(25)).toEqual([0, 0, 0, 1]);
+  });
+
+  it('should return [1, 1, 0, 0] for 6 cents', () => {
+    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
+  });
+
+  it('should return [0, 0, 1, 0] for 10 cents', () => {
+    expect(getCoinCombination(10)).toEqual([0, 0, 1, 0]);
+  });
+
+  it('should return [1, 0, 1, 0] for 11 cents', () => {
+    expect(getCoinCombination(11)).toEqual([1, 0, 1, 0]);
+  });
+
+  it('should return [1, 0, 0, 1] for 26 cents', () => {
+    expect(getCoinCombination(26)).toEqual([1, 0, 0, 1]);
+  });
+
+  it('should return [3, 0, 0, 0] for 3 cents', () => {
+    expect(getCoinCombination(3)).toEqual([3, 0, 0, 0]);
+  });
+
+  it('should return [2, 1, 0, 0] for 6 cents', () => {
+    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
```

Comment:
The expected result for 6 cents should be `[1, 1, 0, 0]` as correctly stated in the test case on line 31. The current expected result `[2, 1, 0, 0]` is incorrect.